src/domain/permissions.spec.js

```diff
@@ -1,25 +1,54 @@
 import { ROLE } from './role.js';
-import sut from './permissions.js';
+import { exchangeUser } from './built-in-users.js';
+import sut, { LIST_EXPORTABLE_CONTENT } from './permissions.js';
+
+const allNonTechnicalPermissions = Object.entries(sut)
+  .filter(entry => (/^[A-Z_]+$/).test(entry[0]))
+  .filter(entry => entry[1] !== LIST_EXPORTABLE_CONTENT)
```

Comment:
Oh in this case you could even have a `allTechnicalPermissions` list with just this guy and use the list both in this filtering and in line 29 (which can also be an iteration over `allTechnicalPermissions`) 😎

Reply:
Unless this is taking it too far, your call anyway 😄

Reply:
I unified the exports of `permissions.js` and reworked this test to go through all exported permissions

Reply:
Ha, cool!